gcp_variant_transforms/testing/test_utils.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
```

Comment:
So, we already have a similar idea in "base_test_case_with_temp_dir.py" ... this is used in "gcp_variant_transforms/libs/vcf_header_parser_test.py".
Do you think you can refactor that test to use this new context manager?
Also, please rename to file to something more specific (test_utils.py ends up being a bag of unrelated stuff over time :) ). Maybe just "temp_dir.py" (i think it should be ok given that we're already in /testing).

Reply:
Got it! Eventually this will class will also be removed from our repo and pulled from beam.